knownprojects_build/_sql/correct_projects.sql

```diff
@@ -148,12 +142,12 @@ DECLARE
     _record_id_match text;
 
 BEGIN
-	<<reassign>>
+	-- <<reassign>>
```

Comment:
Do we need to remove the label in the END LOOP statement?

Reply:
ignore this, I didn't know these are valid pgplsql statement